seeds/pancakeswap/ethereum/pancakeswap_v2_ethereum_trades_samples.csv

```diff
@@ -0,0 +1,6 @@
+blockchain,project,version,block_date,block_time,token_bought_symbol,token_sold_symbol,token_pair,token_bought_amount,token_sold_amount,token_bought_amount_raw,token_sold_amount_raw,amount_usd,token_bought_address,token_sold_address,taker,maker,project_contract_address,tx_hash,tx_from,tx_to,trace_address,evt_index
+ethereum,pancakeswap,2,2023-01-12 0:00,2023-01-12 19:13,WETH,USDC,USDC-WETH,0.1936898575,274.694256,193689857512533280,274694256,275.7814298,0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2,0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48,0x0eae044f00b0af300500f090ea00027097d03000,,0x2e8135be71230c6b1b4045696d41c09db0414226,0x176632fa77a3e328cf904f814888f77ee97c57cde0a014d1dedcab92accfc959,0x05429d641a61eb2c6093905784d2e8c9b5a193c8,0x0eae044f00b0af300500f090ea00027097d03000,,247
+ethereum,pancakeswap,2,2023-01-09 0:00,2023-01-09 3:22,USDC,WETH,USDC-WETH,39.768234,0.03053786191,39768234,30537861913928636,39.75666144,0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48,0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2,0x952454d0915cc41f30dc26f76add170124e3efb4,,0x2e8135be71230c6b1b4045696d41c09db0414226,0xc1f7f412ac0636e914b2e55e21ddbb79f313be2d7ca4c5e084db223787c52441,0x952454d0915cc41f30dc26f76add170124e3efb4,0xeff92a263d31888d860bd50809a8d171709b7b1c,,325
+ethereum,pancakeswap,2,2022-12-01 0:00,2022-12-01 11:34,WETH,USDT,USDT-WETH,7.607205635,9627.386202,7.60721E+18,9627386202,9749.851175,0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2,0xdac17f958d2ee523a2206206994597c13d831ec7,0xa8e7437833f4c2f4f36f8e4559db91bea951bfa9,,0x17c1ae82d99379240059940093762c5e4539aba5,0x70cb37c8e8a566b850d1475606b3c4b64adf51cca9bb9e32645344b93c99c8f9,0x89eed9db3903e8c17bceb5f03d7a10cde8733c62,0xa8e7437833f4c2f4f36f8e4559db91bea951bfa9,,17
```

Comment:
Hi @chef-seaweed, we can move the seed to [dex_trades_seed.csv](https://github.com/duneanalytics/spellbook/blob/main/seeds/dex/trades/dex_trades_seed.csv) for consistency.

Reply:
Hi @Hosuke,thanks for the advice.  I've moved the sample data to [dex_trades_seed.csv](https://github.com/duneanalytics/spellbook/blob/main/seeds/dex/trades/dex_trades_seed.csv) and removed [pancakeswap_v2_ethereum_trades_samples.csv](https://github.com/duneanalytics/spellbook/pull/2806/commits/21f90a855e472114820b80c2a509e19cac744a6d#diff-5910b5426129647e37d4628008bac9a15dd61f5daf01381e05808f32d739d20d). Also would like to what else need to be done to make pancakeswap_v2_ethereum_trades data included in dex.trades? Thank you.